### PR TITLE
Add new args --ignore-list and --ignore-no-bb

### PR DESCRIPTION
**Pull request by danleyb2**
--ignore-list : Allows configuring a Blur ignore list on a text file
--ignore-no-bb : When specified, Won't blur results with no vehicle bounding box

### CODE REVIEW

File: plate_recognition.py [hunk 1]
```diff
@@ -88,8 +89,16 @@ def recognition_api(fp,
     return response.json(object_pairs_hook=OrderedDict)
 
 
-def blur(im, blur_amount, api_res):
+def blur(im, blur_amount, api_res, ignore_no_bb=False, ignore_list=None):
     for res in api_res.get('results', []):
+        if ignore_no_bb and 'vehicle' not in res:
```
Comment (marcbelmont) [inline plate_recognition.py:94]: In both cases, I should do a `continue` else you are no processing the other elements from `api_res['results']`.

File: number_plate_redaction.py [hunk 2]
```diff
@@ -148,6 +160,15 @@ def custom_args(parser):
         '--save-blurred',
         action='store_true',
         help='Blur license plates and save image in filename_blurred.jpg.')
+    parser.add_argument(
```
Comment (marcbelmont) [inline number_plate_redaction.py:163]: Let's use a `parser.add_argument('--ignore-regexp', action='append'...)` instead.

**Review by marcbelmont — COMMENTED** [review]
- [x] update readme
- [ ] remove ignore-list-sample.txt

File: plate_recognition.py (hunk 1 above)
Comment (danleyb2) [reply plate_recognition.py:94]: didn't see this. It's okay now

File: plate_recognition.py [hunk 3]
```diff
@@ -88,8 +89,20 @@ def recognition_api(fp,
...
```
Comment (marcbelmont) [inline plate_recognition.py:94]: `vehicle` is always in res. If no vehicle is detected, `vehicle.score` is 0.

**Review by marcbelmont — APPROVED** [review]

**Pull request merged**